README.adoc

```diff
@@ -0,0 +1,69 @@
+== funops-cli
+
+A library that provides helpers and DRYer import possibilities
+and enhances the following battled tested libraries alread on
```

Comment:
too many and's. also 'already' has a y, unless alread is some haskell thing ive never heard of that makes libraries have laser beams, which is awesome if so.